metrics/cli/lib/common/constants/config_constants.dart

```diff
@@ -0,0 +1,8 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+/// A class that holds the constants for the config.
+class ConfigConstants {
```

Comment:
Does this name fit a bit better? 
```suggestion
class MertricsConfigConstants {
```